src/main/java/net/glowstone/block/itemtype/ItemFish349.java

```diff
@@ -0,0 +1,49 @@
+package net.glowstone.block.itemtype;
+
+import net.glowstone.entity.GlowPlayer;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.potion.PotionEffectType;
+
+public class ItemFish349 extends ItemFood {
```

Comment:
Class name could be more descriptive (not including magic numbers), how about ItemFishRaw?